scripts/base/protocols/rdp/main.zeek

```diff
@@ -280,3 +281,7 @@ event successful_connection_remove(c: connection) &priority=-5
 		write_log(c);
 		}
 	}
+
+event rdpeudp_established(c: connection) {
+	add c$service["rdpeudp"];
```

Comment:
Rather than set this directly, think we should let the existing DPD framework scripts handle it like they generally do for every protocol.  The way it works is by watching for `protocol_confirmation` / `protocol_violation` events, and you get those generated within your analyzer by calling the `ProtocolConfirmation()` / `ProtocolViolation()` methods at points where it is reasonably sure it's trying to parse the right/wrong thing.

Reply:
Again, my ignorance shines. I didn't know ProtocolConfirmation() did that. Fixed!